src/utilities/strategy-helpers/deployment-helper.ts

```diff
@@ -115,14 +115,14 @@ async function checkManifestStability(kubectl: Kubectl, resources: Resource[]):
 async function annotateAndLabelResources(files: string[], kubectl: Kubectl, resourceTypes: Resource[], allPods: any) {
     const workflowFilePath = await getWorkflowFilePath(TaskInputParameters.githubToken);

```

Comment:
Is the workflow expected to fail if any of these extra HTTP calls fail?

Reply:
No. If the HTTP call fails it will go ahead with the value of` GITHUB_WORKFLOW` variable